### PR TITLE
Add SearchBox to Cardigan

### PR DESCRIPTION
**Pull request by davidpmccormick**
screen shot 2018-11-19 at 10 24 23

### CODE REVIEW

**Comment by jamesgorrie** [comment]
As this is going to be emitting events, do we want to test it?

File: common/views/components/SearchBox/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Purpose
+
+To provide a means for users to find what they're looking for.
+
+A form containing a text input and a button.
```
Comment (jamesgorrie) [inline common/views/components/SearchBox/README.md]: What does this sentence mean?

Comment (davidpmccormick) [reply common/views/components/SearchBox/README.md]: Describing the implementation details. Do you think it would be better without it?

Comment (jamesgorrie) [reply common/views/components/SearchBox/README.md]: Probably as those are likely to change.
And it doesn't help with making any further implementation details, e.g. when to use it etc.

Personally keeping docs abstract and idealogical means you probably don't need to update them often, and they don't risk going out of date.

Comment (davidpmccormick) [reply common/views/components/SearchBox/README.md]: yep, makes sense

**Comment by jamesgorrie** [comment]
> As this is going to be emitting events, do we want to test it?

Happy for another PR on this, might be good to pair, as I'd like to rethink how the searchbox affects the results list as it's all quite couply at the moment (albeit simple though).

**Pull request merged**